spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/LoggerTaskRepository.java

```diff
@@ -21,6 +21,8 @@
 import org.slf4j.LoggerFactory;
 
 /**
+ * Default implementation for a task repository that will log the task information.
```

Comment:
Can we leave out the word Default here?  Whether or not it's the default really isn't the implementation's concern (and we'll forget to update that if it changes :)